packages/catalog-model/src/kinds/relations.ts

```diff
@@ -33,7 +33,9 @@ export const RELATION_OWNER_OF = 'ownerOf';
  * A relation with an API entity, typically from a component or system
  */
 export const RELATION_CONSUMES_API = 'consumesApi';
+export const RELATION_API_CONSUMED_BY = 'apiConsumedBy';
 export const RELATION_PROVIDES_API = 'providesApi';
+export const RELATION_API_PROVIDED_BY = 'apiProvidedBy';
```

Comment:
this seems like an omission on our part - we mixed up the two concepts of providing and consuming

Reply:
No provides/consumes are the two different directions 😅, what's the difference between `apiProvidedBy` and `consumesApi`? :grin:

Reply:
C1 <-> A <-> C2

- `A.apiProvidedBy.C1` + `C1.providesApi.A`: `C1` has a listening socket that you can make `A` requests on
- `C2.consumesApi.A` + `A.apiConsumedBy.C2`: `C2` makes fetch requests to services that provide `A` (possibly `C1`)

Reply:
Ah yeah did not consider that there's an API inbetween, you're totally right x)

Reply:
I think we might have to rename the `implementsApis` in the component entity in the future. A better name would be `providesApis`? This would be a breaking change similar to the ancestors and descendants?

Later we can add the same for `consumesApis`. We can also support the "complex" syntax for APIs that work without manually defining an entity, but that is something we can introduce later without a breaking change (see https://github.com/backstage/backstage/pull/2840#issuecomment-713835013)

Reply:
Mm, that's a breaking change for sure, but we could support both in the time between deprecation and deletion. You suggest this mainly from a language consistency standpoint right?

Reply:
Supporting both should be easy. Yes, for consistency only